### DISCUSSION TITLE
如何设置每页顶端和尾端自动空行

### DISCUSSION THREAD

**Discussion by Gilles1551** [sjtug/SJTUThesis#822] [提问] [answered]
如题，毕设要求要顶端和尾端空行，但是目前的模板编译出来并没有空行

**Comment by skyzh** [comment, marked as answer]
哪里的要求？另外如果要空行，可以调整 headsep 选项（即页眉与正文的间距）

**Reply by skyzh** [reply]
https://github.com/sjtug/SJTUThesis/blob/d7a169b354a071839e66980b4fb61930e2f96028/texmf/tex/latex/sjtuthesis/sjtuthesis-undergraduate.ltx#L33
